azure-servicebus/README.rst

```diff
@@ -1,222 +1,83 @@
-Microsoft Azure SDK for Python
-==============================
+Microsoft Azure Service Bus SDK for Python
+==========================================
 
-This is the Microsoft Azure Service Bus Runtime Client Library.
+This is the Microsoft Azure Service Bus Client Library.
+This package has been tested with Python 2.7, 3.4, 3.5, 3.6 and 3.7.
 
-This package has been tested with Python 2.7, 3.3, 3.4 and 3.5.
+Microsoft Azure Service Bus supports a set of cloud-based, message-oriented middleware technologies including reliable message queuing and durable publish/subscribe messaging.
 
-For a more complete set of Azure libraries, see the `azure <https://pypi.python.org/pypi/azure>`__ bundle package.
+* `SDK source code <https://github.com/Azure/azure-sdk-for-python/tree/master/azure-servicebus>`__
+* `SDK reference documentation <https://docs.microsoft.com/python/api/overview/azure/servicebus/client?view=azure-python>`__
 
 
-Compatibility
-=============
+What's new in v0.50.0?
+----------------------
 
-**IMPORTANT**: If you have an earlier version of the azure package
-(version < 1.0), you should uninstall it before installing this package.
+As of version 0.50.0 a new AMQP-based API is available for sending and receiving messages. This update involves **breaking changes**.
+Please read `Migration from 0.21.1 to 0.50.0 <#migration-from-0211-to-0500>`__ to determine if upgrading is
+right for you at this time.
 
-You can check the version using pip:
+The new AMQP-based API offers improved message passing reliability, performance and expanded feature support going forward.
+The new API also offers support for asynchronous operations (based on asyncio) for sending, receiving and handling messages.
 
-.. code:: shell
-
-    pip freeze
-
-If you see azure==0.11.0 (or any version below 1.0), uninstall it first:
-
-.. code:: shell
-
-    pip uninstall azure
-
-
-Features
-========
-
--  Queues: create, list and delete queues; create, list, and delete subscriptions; send, receive, unlock and delete messages
--  Topics: create, list, and delete topics; create, list, and delete rules
--  Event Hubs: create and delete event hubs; send events
+For documentation on the legacy HTTP-based operations please see `Using HTTP-based operations of the legacy API <https://docs.microsoft.com/python/api/overview/azure/servicebus?view=azure-python#using-http-based-operations-of-the-legacy-api>`__.
 
 
-Installation
-============
+Prerequisites
+-------------
 
-Download Package
-----------------
+* Azure subscription - `Create a free account <https://azure.microsoft.com/free/>`__
+* Azure Service Bus `namespace and management credentials <https://docs.microsoft.com/azure/service-bus-messaging/service-bus-create-namespace-portal>`__
```

Comment:
Looks like the **Installation** header needs to be added back in above the `pip install` block (currently looks like its in the prerequisites).